src/content/docs/new-relic-solutions/get-started/glossary.mdx

```diff
@@ -133,6 +133,13 @@ Whether you're considering New Relic or you're already using our capabilities, t
     For an explanation of how basic alerts concepts are related, see [Concepts and workflow](/docs/alerts-applied-intelligence/overview/#concepts-terms).
   </Collapser>
 
+    <Collapser
+    id="alerts"
+    title="Alerts"
+  >
+    [New Relic Alerts](/docs/alerts-applied-intelligence/overview) helps you find, troubleshoot, and resolve problems more quickly. Specifically, it's a hybrid machine learning engine that reduces alert noise, correlates incidents, and automatically detects anomalies.
```

Comment:
Where did this come from?

Reply:
It was applied intelligence, but I think it makes more sense to delete the entry since it's been fully integrated.